.github/workflows/edisonCI.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    - name: Lint with flake8
+      run: |
+        pip install flake8
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+    - name: Test with pytest
```

Comment:
why are you differentiating between different kinds of flake8 tests? If you agree to code according to the standard, then it should reject every violation. Giving a warning for specific tests confuses the standard.

Reply:
we thought to replace the flake8 part in this file with the python syntax checker as described in isssue #10 is it making more sense although the syntax python checker also uses specific tests of flake8?

Reply:
This code section is generated by GitHub. I wouldn't change it unless there is a good reason for it.

Reply:
OK, so can we close with this PR also issue #10?

Reply:
yes

Reply:
If I understand correctly, the code says for E9,F63,F7,F82 it should generate an error and reject, but for max-complexity=10 or if the code is longer than 127 it should just generate a warning. 
IMO, this will cause confusion because some people might look at the warning and make a change, other people might say, its only a warning, I don't care. If the project cares about max complexity and line length, it should be included as an error and if not, it should not be tested.